### PR TITLE
Scrap hastebin, use wastebin for ;;export instead

### PR DESCRIPTION
**Pull request by OtterBoops**

### CODE REVIEW

File: FredBoat/src/main/java/fredboat/audio/source/PlaylistImportSourceManager.java [hunk 1]
```diff
@@ -128,11 +128,6 @@ public void shutdown() {
 
         switch (serviceName) {
 
-            case "hastebin":
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/audio/source/PlaylistImportSourceManager.java]: Why remove the parsing? This is a breaking change.

File: FredBoat/src/main/java/fredboat/util/TextUtils.java [hunk 2]
```diff
@@ -167,23 +163,11 @@ public static void handleException(String logMessage, Throwable e, Context conte
...
      */
     public static CompletionStage<Optional<String>> postToPasteService(String body) {
-        return postToHastebin(body)
-                .thenApply(key -> Optional.of("https://hastebin.com/" + key))
+        return postToWastebin(body)
+                .thenApply(key -> Optional.of("https://wastebin.party/" + key))
                 .exceptionally(t -> {
-                    log.info("Could not post to hastebin", t);
+                    log.error("Could not post to wastebin either", t);
                     return Optional.empty();
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/util/TextUtils.java]: An error message should be sent in this case

**Comment by OtterBoops** [comment]
Tested with https://hastebin.com/icediruxuc.fredboat
FredBoat logs seem fine when resolving this, but hastebin is down.

**Comment by knight-ryu12** [comment]
tbh I think, move hastebin to 2nd option; and send Warning about hastebin will delete its content.

File: FredBoat/src/main/java/fredboat/audio/source/PlaylistImportSourceManager.java [hunk 3]
```diff
@@ -127,12 +127,11 @@ public void shutdown() {
         String serviceName = serviceNameMatcher.group(1).trim().toLowerCase();
 
         switch (serviceName) {
-
             case "hastebin":
-                m = PasteServiceConstants.HASTEBIN_PATTERN.matcher(identifier);
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/audio/source/PlaylistImportSourceManager.java]: You ruined the indentation in this file

**Pull request merged**